.github/actions/spawn-cluster/action.yaml

```diff
@@ -60,11 +60,16 @@ inputs:
     description: Specific arguments to add to the apply command
     required: false
     default: ""
+  skip-prepare:
+    description: Skip the preparation of the worker
+    required: false
+    default: "false"
```

Comment:
Would be nicer to use a boolean for this, IMO

Reply:
To me, we cannot have such type in composite actions, it will always be casted to string

Reply:
You're right, sorry. That's really crappy though